circle.yml

```diff
@@ -1317,7 +1352,7 @@ jobs:
             # run type checking for each individual package
             - run: yarn lerna run types
             - verify-mocha-results:
-                expectedResultCount: 10
+                expectedResultCount: 14
```

Comment:
We weren't verifying on all of the mocha unit tests previously. This takes care of the ones we weren't verifying + the new package.